components/definitions/chromel.d.ts

```diff
@@ -239,6 +239,7 @@ declare namespace chrome.braveToday {
     addListener: (callback: () => any) => void
   }
   const getHostname: (callback: (hostname: string) => any) => void
+  const getRegionURLPart: (callback: (regionURLPart: string) => any) => void
```

Comment:
Just curious: would make sense at some point to just build the whole URL on the C++ side?

Reply:
yes absolutely. Will do as follow-up. Didn't do it here, as it would have been 2 functions 🤔  It's all moving to c++ at some point anyway, but what you're suggesting is a no-brainer.